linalg/ode.hpp

```diff
@@ -314,6 +316,8 @@ class AdamsMoultonSolver : public ODESolver
    Vector *k;
    Array<int> idx;
    ODESolver *RKsolver;
+   double dt_;
+   bool print;
```

Comment:
See comment for addition of `print` member variable in `AdamsBashforthSolver`....

Reply:
@cjvogl I agree! I was just a bit rusty and did not get it to work.. so went for something simple.
Your code snippet is very much appreciated.

Reply:
Thanks for making the change @IdoAkkerman!